Provide clearer instructions on how to specify target language. (#38786)
huggingface/transformers: docs/source/en/model_doc/madlad-400.md

['Eu amo pizza!']
```

> Note that in the input sentence, "`<2pt>`" is a token determining the target language of the translation. To see all supported language codes and their corresponding tokens:

Reviewer: I would add this sentence before the code example so users know what it is beforehand

```suggestion
Include the target language token in the input sentence. `<2pt>` is the token for Portuguese.
```
